### PR TITLE
Add photon-4 OVA support

### PR DESCRIPTION
**Pull request by kkeshavamurthy**
What this PR does / why we need it:
Add support for building Photon-4 OVAs. 
- Added goss tests
- Added CI
- Made doc updates

Which issue(s) this PR fixes (optional, in fixes #<issue number>(, fixes #<issue_number>, ...) format, will close the issue(s) when PR gets merged): Fixes #
#649 
**Additional context**
Add any other context for the reviewers

### CODE REVIEW

File: images/capi/packer/ova/linux/photon/http/4 [hunk 1]
```diff
@@ -0,0 +1 @@
+3
```
[the remark this answers is not in the document]
Comment (kkeshavamurthy) [reply images/capi/packer/ova/linux/photon/http/4:1]: hehe it does look weird on git. you are right, 4 is linked to 3

```
├── photon
│   └── http
│       ├── 3
│       │   └── ks.json
│       └── 4 -> 3
```

[the remark this answers is not in the document]
Comment (mboersma) [reply images/capi/packer/ova/linux/photon/http/4:1]: Oh, it's a symlink! Somehow I couldn't tell that in the GitHub diff view.

**Review by mboersma — APPROVED** [review]
/lgtm
/approve

**Pull request merged**